src/libs/ReportUtils.ts

```diff
@@ -1201,6 +1201,17 @@ Onyx.connect({
     },
 });
 
+let selfDMReportID: string | undefined;
+Onyx.connectWithoutView({
+    key: ONYXKEYS.SELF_DM_REPORT_ID,
+    callback: (value) => {
+        if (!value) {
+            return;
+        }
+        selfDMReportID = value;
+    },
+});
+
```

Comment:
This key is pushed from OpenApp since around month. @mountiny is it safe to assume that it will be present on all accounts?

Reply:
Got it, I think we can just take this key at this point

Reply:
Actually @shubham1206agra Do you have some plan to update this? Maybe we could leave this out of scope of this PR (removal of `findSelfDMReportID` and just use it as a fallback if the SELF_DM_REPORT_ID is not defined)

Reply:
@mountiny You can leave it for this PR. I will migrate it slowly as there are way too many references.